DropboxMoveFile/dcoder_block.yml

```diff
@@ -0,0 +1,18 @@
+id: DROPBOX_MOVE_FILE
+runFunction: main
+fileName: index.js
```

Comment:
Follow the process to build the block into single executable using @vercel/ncc and then it will create a index.js build file in dist folder. Once done change fileName here to dist/index.js and re test your block.

Reply:
Run `dcoder-cli block run-command:run` and select one with vercel/ncc to build the project in dist folder. Then use dist/index.js here to refer main file.
Also you will need to release a new version on Dcoder